en/chromebook_setup/instructions.md

```diff
@@ -74,7 +74,7 @@ Run:
 ```
 mkdir djangogirls
 cd djangogirls
-python3.6 -mvenv myvenv
+python{{ book.py_version }} -mvenv myvenv
```

Comment:
Is 3.9 really available on Cloud9? https://docs.aws.amazon.com/cloud9/latest/user-guide/sample-python.html suggests it's only 3.6.

Reply:
> Is 3.9 really available on Cloud9?

Good point! Seems like it isn't. Thanks for catching that.

> https://docs.aws.amazon.com/cloud9/latest/user-guide/sample-python.html suggests it's only 3.6.

I've just tried, choosing their "Amazon Linux 2 (recommended)" platform. It comes with Python 3.**7**.9, and it seems like it offers no greater Python versions (at least not through `yum`):
```
$ yum list python3
Loaded plugins: extras_suggestions, langpacks, priorities, update-motd
219 packages excluded due to repository priority protections
Installed Packages
python3.x86_64              3.7.9-1.amzn2.0.1              @amzn2-core
Available Packages
python3.i686                3.7.9-1.amzn2.0.1              amzn2-core 
```
(If that's the right way to ask `yum`, that is. I don't really know that package manager.)

> in various places we just use python3 without the minor version. Perhaps that's also better to use here?

Probably, as it doesn't seem like there are typically several Python 3 versions in the same AWS environment. So that'd be
```suggestion
python3 -mvenv myvenv
```

Reply:
(I've now also tried with the (apparently not anymore recommended?) plattform "Amazon Linux" (without "2"). It comes indeed with Python 3.**6**.12, with no obvious possibility to update to a greater version.)

Reply:
You can find all the current python versions in this comment: https://github.com/DjangoGirls/tutorial/pull/1683#issuecomment-727190424

I assume this is resolved now.